release.yaml

```diff
@@ -19,8 +19,10 @@ rancher-vsphere-csi:
 - 100.3.0+up2.5.1-rancher1
 sriov:
 - 100.0.3+up0.1.0
+- 100.0.4+up0.1.0
 sriov-crd:
 - 100.0.3+up0.1.0
+- 100.0.4+up0.1.0
```

Comment:
You don't need to bump the version in package.yaml or release.yaml. `100.0.3+up0.1.0` is a new version that's being released in 2.6.5.

Reply:
But then, how the user will be able to select `100.0.4+up0.1.0`?

Reply:
@manuelbuil The last sriov version we released is [100.0.2+up0.1.0](https://github.com/rancher/charts/tree/release-v2.6/charts/sriov) which is the latest in the release-v2.6 branch, but in dev-v2.6 the latest is [100.0.3+up0.1.0](https://github.com/rancher/charts/tree/dev-v2.6/charts/sriov). I added the latter to fix this [issue](https://github.com/rancher/rancher/issues/36069), and to do so I had to bump the version which then created a new chart. If you bump it again we would end up with 2 new charts to be released, but we only want to release one chart per Rancher release. All you have to do to avoid that is revert the version bump in your PR so that the new 100.0.3+up0.1.0 includes both my fix and your update. Once released, the user will see the 100.0.3+up0.1.0 as the new chart, and can then select it to upgrade from 100.0.2+up0.1.0.

Reply:
I see. Thanks a lot for the explanation!